content/en/seachat/seachat-manual/03-add-knowledge/02-spreadsheet-upload.md

```diff
@@ -38,40 +38,47 @@ By clicking on the upload button, you can submit to your agent choosing from var
 > :rotating_light: **Note** :rotating_light:
 > 
 > Before uploading the file, check if the files meet the two following requirements:
-> - Table must be headers in the first row.
-> - The content in any row may not exceed 2000 tokens.
+> - Your spreadsheet table must use the first row as the header. If your header is the first column, please [transpose your table](https://support.microsoft.com/en-us/office/transpose-rotate-data-from-rows-to-columns-or-vice-versa-3419f2e3-beab-4318-aae5-d0f862209744) first.
+> - The content in any row may not exceed 2000 tokens. If your row exceeds this limit, please contact us!
 > 
-> Check "Upload Guidelines" for more information.
+> Check **Upload Guidelines** for more information.
 
-## Step 4: Before Submission
+## Before Submission
 SeaChat allows users to upload in bulk. You can see the status of each uploading file in the section below the drag-and-drop zone.
```

Comment:
please be more specific: 
1. one can upload more than one excel
2. each excel can contain more than one worksheet